assets/css/nav/_left_nav.scss

```
@@ -10,14 +10,12 @@
   .c-left-nav__icon {
     padding-right: 0.75rem;
   }
+
+  transition: all 0.3s;
```

suggestion: when I try this in Safari (the only browser I can find with a `prefers-reduced-motion` dev-tools toggle), the animation still plays, I believe this should only enable this animation if the user doesn't prefer reduced motion.

```suggestion
  @media not (prefers-reduced-motion: reduce) {
    transition: all 0.3s;
  }
```

---

https://github.com/mbta/skate/pull/2403/commits/3b3ba30a89109f8e7c0566613623dcabae916eb6